buzsaki_lab_to_nwb/fujisawa_code/fujisawamiscdatainterface.py

```diff
@@ -0,0 +1,108 @@
+"""Authors: Cody Baker and Ben Dichter."""
+import os
+import numpy as np
+from pathlib import Path
+from scipy.io import loadmat
+import warnings
+
+from nwb_conversion_tools.basedatainterface import BaseDataInterface
+from pynwb import NWBFile
+from pynwb.file import TimeIntervals
+from pynwb.behavior import SpatialSeries, Position
+from hdmf.backends.hdf5.h5_utils import H5DataIO
+
+from ..neuroscope import get_events, check_module
+
+
+class FujisawaMiscInterface(BaseDataInterface):
+    """Primary data interface for miscellaneous aspects of the FujisawaS dataset."""
+
+    @classmethod
+    def get_source_schema(cls):
+        return dict(properties=dict(folder_path="string"))
```

Comment:
similar problem here, regarding the json schema

Reply:
NWBConverters should return valid json schemas but I don't know if that's necessary for DataInterfaces